website/docs/r/satellite_dns_register.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+subcategory: "Satellite"
+layout: "ibm"
+page_title: "IBM : satellite_dns_register"
+description: |-
+  Managed satellite dns register.
+---
+
+# ibm\_satellite\_dns\_register
+
+Provides a resource to register public ip address to satellite dns records. This allows satellite dns register to be created, updated and deleted.
+
+## Example Usage
+
+```hcl
+resource "satellite_dns_register" "satellite_dns" {
+  location = "satellite-ibm"
+  ips      = ["52.116.125.50","169.62.17.178","169.63.178.155"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `ips` - (Required, Forces new resource, List) Public IP address of satellite location DNS records.
+* `location` - (Required, Forces new resource, string) The name or ID of the Satellite location.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The unique identifier of the satellite_dns_register.
+
+## Import
+
+You can import the `ibm_satellite_dns_register` resource by using `location`.
+The `location` property can be formed from 
+
+```
+
```

Comment:
Add The import functionality is not supported for this resource.

Reply:
updated the doc